docs/learning/display/Classification.md

```diff
@@ -7,7 +7,7 @@ Spatial classification and planar clip masking are related ways of using one set
 Spatial classification uses one or more design models to contextualize a reality model by associating portions of the reality model with elements in the design models. Two types of classification are supported:
 
 - Planar classification: The design models are projected onto a plane. The region of the reality model that intersects the volume extruded from an element's footprint become classified by that element.
-- Volume classification: Regions of the reality model that intersect the 3d volume of an element become classified by that element.
+- Volume classification: Regions of the reality model that intersect the 3d volume of an element become classified by that element.  Geometry used for volume classification must be fully closed, non-intersecting volumes.
```

Comment:
Please add the same clarification to the [learning article](https://github.com/iTwin/itwinjs-core/blob/master/docs/learning/display/Classification.md).

Reply:
I thought that's what I just did.  I went to the learning article and used the edit command and after finishing it created this PR for it.

Reply:
I'm sorry -- I meant the [API documentation](https://github.com/iTwin/itwinjs-core/blob/aca5d1769dc2eb12e83bedf49e7e5804b9c33416/core/common/src/SpatialClassification.ts#L144).

Reply:
OK, Done.